dydx/dydxPresenters/dydxPresenters/_v4/MarketInfo/dydxMarketInfoViewBuilder.swift

```diff
@@ -26,23 +26,21 @@ public class dydxMarketInfoViewBuilder: NSObject, ObjectBuilderProtocol {
 }
 
 private class dydxMarketInfoViewController: HostingViewController<PlatformView, dydxMarketInfoViewModel> {
+    private var hidePredictionMarketsNotice: Bool {
+        SettingsStore.shared?.value(forKey: dydxSettingsStoreKey.hidePredictionMarketsNoticeKey.rawValue) as? Bool ?? false
```

Comment:
Nit: Since settings read/write is shared among multiple components, might want to consolidate it into a separate class (see `dydxFavoriteStore`)